config/prod/kelsey-ami.yaml

```diff
@@ -0,0 +1,49 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-v8.j2
+stack_name: kelshmo-cmc
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "SysBio"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "cmc"
+  # The resource owner
+  OwnerEmail: "kelsey.montgomery@sagebase.org"
+
+  # Default account settings, leave alone unless you plan to override
+  VpcName: "{{stack_group_config.default_vpc}}"
+  KeyName: "{{stack_group_config.default_key_pair}}"
+  AMIId: "{{stack_group_config.default_ami}}"
```

Comment:
test failed, says this definition is a duplicate.  Please comment this one out since your specifying your own AMI below.